alf/algorithms/ppo_loss.py

```diff
@@ -64,10 +66,12 @@ def __init__(self,
                 Q values and returns the loss for each element of the batch.
             use_gae (bool): If True, uses generalized advantage estimation for
                 computing per-timestep advantage. Else, just subtracts value
-                predictions from empirical return.
+                predictions from empirical return. This is ignored by PPOLoss2
+                (Alwasy assumed to be True)
```

Comment:
Maybe add one more sentence describing the reason of ignoring, e.g., because the advantages are computed in another place using the old policy.

Reply:
use_gae removed